src/config/env.c

```diff
@@ -333,7 +333,7 @@ bool __attribute__((nonnull(1,2,3))) readEnvValue(struct conf_item *conf_item, s
 			unsigned int val = 0;
 			if(sscanf(envvar, "%u", &val) == 1 && val <= UINT16_MAX)
```

Comment:
I was going to ask if we should be using `%hu` here but not actually needed as we have the manual check `val <= UINT16_MAX`